infrastructure/stacks/performance-pipelines/events.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_cloudwatch_event_rule" "stress_test_cron" {
+  name                = "${var.project_id}-${var.environment}-stress-test-trigger"
+  description         = "Trigger performance pipeline every Monday at midnight"
+  schedule_expression = "cron(0 0 ? * 2 *)" # every Monday at midnight
+}
+
+resource "aws_cloudwatch_event_target" "trigger_stress_test_pipeline" {
+  rule      = aws_cloudwatch_event_rule.stress_test_cron.name
+  target_id = "TriggerStressTestPipeline"
+  arn       = aws_codepipeline.stress_test_codepipeline.arn
+  role_arn  = aws_iam_role.performance_rule_role.arn
+}
+
+resource "aws_cloudwatch_event_rule" "load_test_cron" {
+  name                = "${var.project_id}-${var.environment}-load-test-trigger"
+  description         = "Trigger performance pipeline every Tuesday at midnight"
+  schedule_expression = "cron(0 0 ? * 3 *)" # every Tuesday at midnight
```

Comment:
can we run this one every day at midnight and the other one every other day at 3am or something so that they dont clash and we get some output daily?